UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartTraumaDamage.cs

```diff
@@ -122,11 +140,51 @@ public void ApplyTraumaDamage(float tramuaDamage, TraumaticDamageTypes damageTyp
 				if (damageType == TraumaticDamageTypes.PIERCE) { currentPierceDamage += MultiplyTraumaDamage(tramuaDamage); }
 				CheckCutSize();
 			}
-			//Burn damage checks for it's own armor damage type.
+			//Burn and blunt damage checks for it's own armor damage type.
 			if (damageType == TraumaticDamageTypes.BURN)
 			{
 				TakeBurnDamage(MultiplyTraumaDamage(tramuaDamage));
 			}
+
+			if (damageType == TraumaticDamageTypes.BLUNT)
+			{
+				if (DMMath.Prob(SelfArmor.Melee * 100) == false)
+				{
+					TakeBluntDamage(tramuaDamage);
+				}
+			}
+		}
+
+		public void TakeBluntDamage(float damage)
+		{
+			void TakeBluntLogic(BodyPart bodyPart)
+			{
+				bodyPart.health -= damage;
+				bodyPart.CheckIfBroken(true);
+			}
+
+			foreach (ItemSlot slot in OrganStorage.GetIndexedSlots())
+			{
+				if (slot.IsEmpty) { return; }
+				if (slot.Item.gameObject.TryGetComponent<BodyPart>(out var bodyPart))
+				{
+					if (bodyPart.CanBeBroken) { TakeBluntLogic(bodyPart);}
+				}
+			}
+		}
+
+		public void CheckIfBroken(bool announceHurtDamage = false)
+		{
+			if (!CanBeBroken) { return; }
```

Comment:
```suggestion
			if (CanBeBroken == false) { return; }
```

Reply:
no, Because you only need to report if a bone is broken; not if it's cut or pierced.

Reply:
WAIT WRONG SCRIPT

Reply:
wot?

Reply:
brain fart ignore what i said aaaaaaaaa